redisson/src/main/java/org/redisson/remote/BaseRemoteProxy.java

```diff
@@ -44,7 +44,7 @@
  */
 public abstract class BaseRemoteProxy {
 
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final Logger log = LoggerFactory.getLogger(BaseRemoteProxy.class);

```

Comment:
this needs to remain unchanged or be added to all inheritance classes with concrete class, not BaseRemoteProxy.